components/ModEditSubmissionForm.vue

```diff
@@ -1,4 +1,24 @@
 <template>
+    <Modal
+        v-if="modalStore.isOpen"
+        class=" min-h-20 min-w-20 border-8"
+    >
+        <p>Are you sure you want to leave? You have unsaved changes!</p>
```

Comment:
It's a good thing to avoid adding raw strings, because we may forget to localize them later. Let's add this line to the localization files!

Reply:
Sorry for the confusion. If you take a look at the most recent commit you will find the raw strings aren't there and we have updated the en locale file, too.